packages/lit-element/CHANGELOG.md

```diff
@@ -24,20 +24,20 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 * [Breaking] Decorators are no longer exported from the `lit-element` module. Instead import any decorators you use from `lit-element/decorators/*`.
 * [Breaking] `lit-html` has been updated to 2.x. Note, shady-render support has been removed. Import the shady-render package to support Shady DOM.
 
+### Fixed
+* Fixes exceptions when parsing attributes from JSON ([#722](https://github.com/Polymer/lit-element/issues/722)).
+* Fixes issue with combining `static get properties` on an undefined superclass with `@property` on a subclasss ([#890]https://github.com/Polymer/lit-element/issues/890));
```

Comment:
Do we consider both of these fixes non-breaking? (wasn't sure if there was a reason we're doing them in lit-next)

Reply:
Yes, not critical to go in 3.x but simpler.